express-get-json/server.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express';
+
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const getGrades = (req, res, next) => {
+  const json = [];
+  for (const key in grades) {
+    json.push(grades[key]);
+  }
+  next(sendBack(res, json));
```

Comment:
I'm not sure why you use `next` for this?

I'm not actually convinced this works as you intend. The Express spec says that if you call `next` with an argument that is not empty then it is assumed to be an error and subsequent middleware is skipped (until it finds an error handling middleware, which we will discuss later).

Your `sendBack` happens to return `undefined` because it doesn't have an explicit return statement, so you are calling `next` even after sending a response. You usually want to either send a response _or_ call `next`, but not both.

I'm not sure if you were trying to be clever, or if you didn't quite understand how this all works. So I'm going to ask you to rework this before I approve.